DataBinding/Realm.DataBinding.PCL/TypeInfoHelper.cs

```diff
@@ -27,8 +27,7 @@ public static class TypeInfoHelper
         [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:ElementsMustBeDocumented")]
         public static TypeInfo GetInfo<T>() where T : RealmObject
         {
-            RealmPCLHelpers.ThrowProxyShouldNeverBeUsed();
-            return null;
+            return typeof(T).GetTypeInfo();
```

Comment:
I felt it's semantically incorrect to throw pcl exception and tell people to reference the platform specific version when they can't really do that (since we only have platform specific versions for iOS and Android). They shouldn't be using the library anyway, but if they somehow did, we can at least not throw :)